src/component/inspect-request/request-details.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import RequestHeaders from './request-headers.js';
+import FullUrl from '../request/full-url.js';
+
+const {object} = React.PropTypes;
+
+const RequestDetails = (props) => {
+  const {request} = props;
+
+  return <div className="box-body">
+    <section>
+      Request URL:
+      <FullUrl request={request.request} />
+    </section>
+    <section>
+      Request Query Params:
+      <RequestHeaders headers={request.request.query} />
```

Comment:
Not an issue with this PR per se, but we shouldn't show "Request Query Params" if there are none. Should it be in it's own component, so it can return `<div></div>` if necessary?